nexus/src/external_api/http_entrypoints.rs

```diff
@@ -809,6 +810,39 @@ async fn project_instances_delete_instance(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/**
+ * Migrate an instance to a different propolis-server, possibly on a different sled.
+ */
+#[endpoint {
+    method = PUT,
```

Comment:
Hmm, I think you're right and this should be a `POST`. Because, even if it is the same sled, we'll be modifying the Instance record in the DB to point to a new propolis instance.